server/modules/health-metrics/health-metrics.controller.ts

```
@@ -0,0 +1,93 @@
+    }
+  );
+
+  /**
+   * @route   GET /api/v1/metrics/history
+   * @desc    Get historical metrics
+   * @access  Private
+   * @permission get_health_check
+   */
+  public getMetricsHistory = asyncHandler(
+    async (req: AuthenticatedRequest, res: Response): Promise<void> => {
+      const hours = req.query.hours ? parseInt(req.query.hours as string) : 24;
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)